modules/gdscript/gdscript_parser.cpp

```diff
@@ -3504,21 +3505,23 @@ static String _process_doc_line(const String &p_line, const String &p_text, cons
 				from = rb_pos + 1;
 
 				String tag = line.substr(lb_pos + 1, rb_pos - lb_pos - 1);
-				if (tag == "code") {
+				if (tag == "code" || tag.begins_with("code ")) {
```

Comment:
Maybe it would be better here to make a distinction between the initial "tag" and additional parameters, using something like `String.split()`. But then there's `result += "[" + tag + "]";` below so I am not sure.